site/src/components/roadmap/style.module.css

```diff
@@ -0,0 +1,79 @@
+.heading {
+  margin-bottom: 5%;
+  text-align: center;
+}
+
+.heading3 {
+  padding-bottom: 5%;
+}
+
+.spacing {
+  border-radius: 8px;
+}
+
+.cardContainer:global(.saltGridLayout) {
+  margin-top: 50px;
+  display: flex;
+  flex-wrap: wrap; /* Allow items to wrap to the next line */
+  gap: 20px; /* Adjust the gap between columns */
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.content {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.bar {
+  flex: 1;
+  max-width: 400px;
+  width: 100%;
+}
+
+.test {
+  width: "400px";
+  margin-bottom: " 50px ";
+}
+.date {
+  display: inline;
+}
+
+.icon {
+  margin-bottom: -3px;
```

Comment:
negative margins are somewhat painful/fragile when creating library components.. would be keen to know why this is needed.